### DISCUSSION TITLE
Ikea Fyrtur Blinds (E1757) paired with z2m binding to Ikea Remote (E1766) not working?!

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#6111] [Devices]
Hey everyone,

In an attempt trying to avoid opening another issue on this, I wanted to see if someone successfully managed to operate their Ikea FYRTUR blinds with z2m as well as with the included Remote (E1766) through binding? 

I was able to successfully pair the FYRTUR blinds with z2m and I can control them, in the network map, it seems that my router is connected to the included Ikea Zigbee Repeater which in fact connects to the FYRTUR blind and the Remote.

I can control everything and see the click event etc. 

Now when I try to bind the remote to blind (using the new frontend), I first have to unbind the default group (as described on the z2m device page) and then bind the blinds with the switch. The only cluster that will be bound is "Closures" - I think this is fine, but would love to get confirmation on this.
Afterwards unfortunately, nothing moves when I click my remote ...

Was anyone able to successfully get these devices to work together using binding?
I'd love to use my blinds with binding rather than through node-red or something as it would allow smoother control of the two! 

Thanks

**Comment by paulz-gh** [comment]
Hi, I have setup my blinds yesterday and have them bound to remotes.
I have one pair of Fyrtur blinds in one group and bound one remote to that group, and the third KADRILJ  blind is the only device in another group and another remote is bound to that group.

Set up was done running z2m 1.17.0 but upgraded to 1.17.1 this morning.
Using zzh stick as coordinator.

**Reply by ghost** [reply]
Nice and thank you for that answer.

I've been trying to bind the remotes directly to the blind and not to a group. That is, I'm understanding correctly that you bound the remote to a group and not just the device, right?

Can you maybe tell me the clusters that are bound in this case then?

Thanks!

**Reply by paulz-gh** [reply]
According to the zigbee2mqtt web tool one remote has the OnOff LevelCtrl and Closures clusters bound to the group containing 2 blinds ( in addition to PowerCfg bound to the coordinator). 
But the other remote has nothing bound except the coordinator bind, but it still works. 
So it looks like the web tool does not show all information.
I don't know if there is another way to find any bindings...

 Paul

**Reply by ghost** [reply]
PAUL! Thank you for your reply, I created groups for each one of the blinds (single device in the group) and now the onoff levelctrl and closures are bound and the remotes work!!!
Thank you so much for that hint
